css-dev/burf-base.scss

```
@@ -4,7 +4,8 @@
 
 // dependencies and resets
 @import "normalize/import-now";
-@import "math";
+@import "node_modules/normalize-scss/sass/normalize/import-now";
+@import "node_modules/mathsass/dist/math";
```

I think we want to use the tilde`~` style imports here for consistency. 

```suggestion
@import "~/mathsass/dist/math";
```

---

This works for webpack and CGB, but unfortunately doesn't seem to work with grunt-sass, so I think it has to stay as is.